search/django/weedid/tasks.py

```diff
@@ -8,22 +8,31 @@
 from weedid.models import Dataset
 from weedid.utils import make_upload_entity_fields
 from weedid.notification import upload_notification
-from core.settings import THUMBNAILS_DIR, REPOSITORY_DIR, DOWNLOAD_DIR
+from core.settings import THUMBNAILS_DIR, REPOSITORY_DIR, DOWNLOAD_DIR, UPLOAD_DIR
 from pathlib import Path
+import os
+from shutil import move, rmtree
 
 
 @shared_task
-def submit_upload_task(weedcoco_path, image_dir, upload_id):
+def submit_upload_task(weedcoco_path, image_dir, upload_id, new_upload=True):
```

Comment:
Please add a docstring describing new_upload semantics. The fact that it switches from treating weedcoco data as primary to treating the DB data as primary is a bit quirky and definitely needs documenting